tests/RateLimitErrorTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Stripe;
+
+class RateLimitErrorTest extends TestCase
+{
+    public function testRateLimit()
+    {
+        try {
+            throw new Error\RateLimit(
```

Comment:
This code is essentially testing that the compiler does what's expected. @nelhage gave a [talk about the concept](https://docs.google.com/presentation/d/1QBJ4EntC63FX-R4THZgSFRZNi_R0H7FovN5JfJmgcVA/edit#slide=id.g42d6fffa0_00)

If you want to test this, you should mock out the response so that we'd actually be testing the response parsing. Something similar to https://github.com/stripe/stripe-php/blob/master/tests/TestCase.php#L32-L39